pkgs/development/tools/rover/default.nix

```diff
@@ -41,10 +55,10 @@ rustPlatform.buildRustPackage rec {
   '';
 
   meta = with lib; {
-    description = "A CLI for managing and maintaining graphs with Apollo Studio";
+    description = "A CLI for interacting with ApolloGraphQL's developer tooling, including managing self-hosted and GraphOS graphs.";
     homepage = "https://www.apollographql.com/docs/rover";
     license = licenses.mit;
-    maintainers = [ maintainers.ivanbrennan ];
-    platforms = ["x86_64-linux"];
+    maintainers = [ maintainers.ivanbrennan maintainers.aaronarinder ];
+    platforms = ["x86_64-linux" "aarch64-darwin"];
```

Comment:
This should be `platforms.unix`, or just omit it entirely if it works on both Linux and Darwin. Right now, you're limiting it to just x86_64-linux and aarch64-darwin, when aarch64 and x86_64 versions of those systems.

Reply:
done!